README.md

```diff
@@ -1,98 +1,244 @@
-* [Media Blob Explainer](MediaBlob/explainer.md)
-
-## Navigation
-* [Arbitrary Text Fragments Explainer](Fragments/explainer.md)
-
-## Privacy
-* [Autofill Reauthentication](AutofillReauthentication/explainer.md)
-* [Time-limited Permissions Explainer](TimeLimitedPermissions/Explainer.md)
-* [Principles: Auditing Privacy on the Web](WebPrivacyAuditing/explainer.md)
-
-## Progressive Web Applications
-* [Run on OS Login Explainer](RunOnLogin/Explainer.md)
-* [Cache API Response Metadata Explainer](CacheAPIResponseMetadata/explainer.md)
-* [Install Time Permissions Prompt](InstallTimePermissionsPrompt/Explainer.md)
-* [Title Bar Customization Explainer](TitleBarCustomization/explainer.md)
-* [URL Protocol Handler Registration for PWAs Explainer](URLProtocolHandler/explainer.md)
-* [Version History](VersionHistory/explainer.md)
-
-## Smooth Scrolling
-  * [Impl Threaded Scrollbar scrolling intent-to-implement](https://groups.google.com/a/chromium.org/forum/#!topic/input-dev/6ACOSDoAik4)
-
-## Storage
-* [Storage Access API Explainer](StorageAccessAPI/explainer.md)
-
-## UI
-* [Window segments enumeration API Explainer](Foldables/explainer.md)
-
-## Web App Manifest
-* [ImageResource `color_scheme` Explainer](ImageResource-color_scheme/explainer.md)
-* [Shortcuts Explainer](Shortcuts/explainer.md)
-
-## WebDriver
-* [Bidirectional WebDriver Protocol Explainer](WebDriverRPC/webdriver.md)
-
-## Web Performance Measurement
-* [Frame Timing Use Cases](FrameTiming/use_cases.md)
-
-## 3D / Immersive Experiences
-* [Native GLTF Pre-explainer](https://github.com/immersive-web/proposals/issues/52)
-
-## Withdrawn
-* [Custom Dialog on Close Explainer](Withdrawn/CustomDialogOnClose/explainer.md)
+* [Streamlined Text Input (TSF1)](TSF1/explainer.md) explainer
+* [PenEvents](PenEvents/dev-design.md) design document
+
+### Fonts
+
+* [End User Defined Characters (EUDC)](EUDC/explainer.md) explainer
+
+### HTML
+
+* [HTML Modules](HTMLModules/i2i.md) intent-to-prototype
+* [JSON Modules](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/ojwkySW-bpQ) intent-to-prototype
+* [Synthetic Module Record](SyntheticModules/designDoc.md) design document
+
+### Media
+
+* [Hardware-offloaded Audio Processing](AudioOffload/explainer.md) explainer
+* [Media Cache Reduction](MediaCacheReduction/explainer.md) explainer
+
+### Privacy
+
+* [Auditing Privacy on the Web](WebPrivacyAuditing/explainer.md) a principles document describing our commitment to privacy through the principles of transparency, control, respect, and protection for our users.
+* [Autofill Reauthentication](AutofillReauthentication/explainer.md) explainer
+
+### User Interaction
+
+* [Impl Threaded Scrollbar scrolling](https://groups.google.com/a/chromium.org/forum/#!topic/input-dev/6ACOSDoAik4) intent-to-implement
+
+### Web Performance Measurement
+
+* [Frame Timing](FrameTiming/use_cases.md) use cases
+
+# Withdrawn
+
+When at first you don't succeed... don't give up! We're no longer pursuing
```

Comment:
Instead of "..." use the actual ellipses: "…"

Reply:
👍